sources/packages/backend/libs/sims-db/src/entities/msfaa-number.model.ts

```diff
@@ -103,6 +104,12 @@ export class MSFAANumber extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   student: Student;
+  /**
+   * ApplicationId related to this MSFAA.
+   */
+  @RelationId((msfaaNumber: MSFAANumber) => msfaaNumber.referenceApplication)
```

Comment:
Same as https://github.com/bcgov/SIMS/pull/2010/files#r1230299287

Reply:
Removed @Column and kept @RelationId as it is as per discussion with @andrewsignori-aot and @ann-aot